portal-api/api-tests/v1/gef/facilities.api-test.js

```diff
@@ -366,6 +366,39 @@ describe(baseUrl, () => {
       const { status } = await as(aMaker).put({}).to(`${baseUrl}/doesnotexist`);
       expect(status).toEqual(204);
     });
+
+    it('should update the coverStartDate and coverEndDate in the right format set to midnight', async () => {
+      const { details } = newFacility;
```

Comment:
Can you please add following test scenario:

1. Leap year date
2. Date with year `9999` and `10000`

Reply:
done